### PR TITLE
Added Django>1.6 migrations

### PR DESCRIPTION
**Pull request by ZuluPro**

### CODE REVIEW

**Comment by Fantomas42** [comment]
Hello,

maybe a better solution is to adapt the migrations for the new migration system bundled within Django.

**Comment by ZuluPro** [comment]
Oh, I didn't see it was made with South. Gimme some times I'll adapt it.

**Comment by Fantomas42** [comment]
Cool !

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+script:
+    - coverage run manage.py test
+
+after_success:
+    - ./bin/coveralls
+
+# notifications:
+#     irc:
+#         - "irc.freenode.org#django-blog-zinnia"
```
Comment (ZuluPro) [inline .travis.yml]: Added `travis.yml` for test across different versions of Python and Django.
There's no tests, but migrations are launched.

**Comment by ZuluPro** [comment]
Done !
I've moved South's migrations as explained here: https://docs.djangoproject.com/en/1.8/topics/migrations/#libraries-third-party-apps

**Comment by ZuluPro** [comment]
@Fantomas42 Any news ?
I need a ne release for update my blog.

**Comment by Fantomas42** [comment]
@ZuluPro zinnia-wysiwyg-tinymce v1.3 released !

Thanks for your work.

**Pull request merged**